scripts/jsonrpcproxy.py

```diff
@@ -221,6 +238,14 @@ def process(self, request):
 
         return response
 
+    def run(self):
+        self.conn = get_ipc_connector(self.backend_address)
+        self.conn.check_connection(timeout=10.0)
```

Comment:
There is a `BUFSIZE=32` in the top. Maybe move this as a configuration value up there too?

Reply:
Also suggest to bump the `VERSION` there.

Reply:
Bumped to 0.2